app/assets/stylesheets/custom.scss

```diff
@@ -148,10 +148,10 @@ a {
 }
 
 .profile-header {
-    background-color: white;
-    padding-top: 60px;
-    padding-left: 50px;
-    padding-bottom: 40px;
+  background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.